src/index.ts

```diff
@@ -30,16 +40,18 @@ export class MhDynamoDbClient {
     options?: Options,
     dynamoDbClientConfig: DynamoDBClientConfig = {}
   ) {
+    this.options = this.mergeOptions(defaultOptions, options);
+
     const dynamoDb = options?.awsClientCapture
       ? options.awsClientCapture(new DynamoDB(dynamoDbClientConfig))
       : new DynamoDB(dynamoDbClientConfig);
+
     this.client = DynamoDBDocumentClient.from(dynamoDb);
-    this.options = options;
   }
 
   async getItem<T>(args: GetCommandInput): Promise<T> {
     const response = await this.client.send(new GetCommand(args));
-    return response.Item as T;
+    return this.sanitizeItem(response.Item as T);
```

Comment:
Should it be possible to override set options here again? If I for some reason use the same client but for this specific call I want to use different/altered options?

```ts
  async getItem<T>(args: GetCommandInput, options?: Options['fetchOptions']): Promise<T> {
    const response = await this.client.send(new GetCommand(args));
    this.sanitizeItem(response.Item as T, options);
  }
```

sanitizeItem should prio options in parameter.

Opens up new questions:
- Might be different part of options available depending on function; `getItem` options might be of type Options['fetchOptions'] | undefined for example?
- Should the options sent directly **override** _or_ **merge** with class options?